cfn-templates/cid-cfn.yml

```diff
@@ -68,11 +69,13 @@ Metadata:
       Suffix:
         default: "Suffix - Please do not change"
       QuickSightDataSetRefreshSchedule:
-        default: "QuickSight DataSet Refresh Schedule"
+        default: "QuickSight DataSet Refresh Schedule. Must be a valid cron or empty. If empty refresh will be disabled."
```

Comment:
Maybe default to daily? It's the most common use case. Or another option is we could provide cron example for daily here

Reply:
default is 
`    Default: cron(0 4 * * ? *)`
https://github.com/aws-samples/aws-cudos-framework-deployment/blob/005e9c9c4f0532214c9de5b0bdecf61fa7dc4085/cfn-templates/cid-cfn.yml#L104